.travis.yml

```diff
@@ -13,15 +13,14 @@ notifications:
       - 130s@2000.jukuin.keio.ac.jp
 env:
   global:
-    - ROS_DISTRO=dashing
+    - ROS_DISTRO=foxy
     - ROS_REPO=ros
     - UPSTREAM_WORKSPACE=geometric_shapes.repos
     - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter"
     - WARNINGS_OK=false
   matrix:
-    - TEST="clang-format, ament_lint"
-    - ROS_DISTRO=dashing
-    - ROS_DISTRO=eloquent  TEST=code-coverage
+    - TEST="ament_lint"  # TODO(henningkayser): re-enable clang-format
```

Comment:
clang format does not work yet for ROS 2? Is this because there isn't an agreement on version number yet?

Reply:
It's not about the version number, we need a new configuration that should be applied to all repos. clang-format-3.9 is not available on Ubuntu 20.04 so I disabled it in CI. We could temporarily run clang-format-3.9 on the eloquent image (like https://github.com/ros-planning/moveit2/pull/227/commits/7aea7a713c66b13ca9c47fe4dc5c098512a6886d)